stream.js

```diff
@@ -92,11 +92,12 @@ page.open(opts.url, function (status) {
 	window.setTimeout(function () {
 		if (opts.hide) {
 			page.evaluate(function (els) {
-				els.forEach(function (el) {
+				var hideDomElement = function (el) {
```

Comment:
See in the last test run,

https://travis-ci.org/kevva/screenshot-stream/jobs/98161996

```
/home/travis/build/kevva/screenshot-stream/stream.js
  96:53  warning  Too many nested callbacks (5). Maximum allowed is 4  max-nested-callbacks
```

This warning is provided by `xo`.

This change fixes that warning.